<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqOpenVRDockPanel</class>
 <widget class="QWidget" name="pqOpenVRDockPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="multisamples">
        <property name="toolTip">
         <string extracomment="Whether or not to use multisampled framebuffers. Using multisamples can reduce flickering but currently does not work with volume rendering."/>
        </property>
        <property name="text">
         <string>Use MultiSamples</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="baseStationVisibility">
        <property name="text">
         <string>Base Station Visibility</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendToOpenVRButton">
        <property name="text">
         <string>Send to OpenVR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="attachToCurrentViewButton">
        <property name="text">
         <string>Attach to Current View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showVRViewButton">
        <property name="toolTip">
         <string>Display a window showing a stabilized view of what the VR heasdet is showing</string>
        </property>
        <property name="text">
         <string>Show VR View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportLocationsAsViewButton">
        <property name="toolTip">
         <string extracomment="This option will export the saved locations, their geometry, txtures, and properties into a form that can be used in MineView. The result is stored in a pv-view directory. "/>
        </property>
        <property name="text">
         <string>Export Locations as a View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportLocationsAsSkyboxesButton">
        <property name="toolTip">
         <string extracomment="Export the saved locations from VR as skyboxes suitable for viewing with vtk.js. This option will produce a pv-skybox folder that can be zipped up for use by ParaView Glance or vtk.js."/>
        </property>
        <property name="text">
         <string>Export Locations as Skyboxes</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="cropThickness">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string extracomment="Thickness to use for new thick crops. A value of 0 defaults to automatic thickness."/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cropThicknessLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string extracomment="When using thick cropping planes this specifies the thickness of those planes."/>
          </property>
          <property name="text">
           <string>Crop Thickness</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="scalarFieldLabel">
          <property name="text">
           <string>Editable Field</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editableField">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string extracomment="What scalar field can be edited in VR. This is an advanced feature still in development. You must specify field values below for this to work and the editable field specified must be a scalar array present in your data."/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="fieldValuesLabel">
          <property name="text">
           <string>Field Values</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fieldValues">
          <property name="toolTip">
           <string extracomment="The options available when editing a field in VR. These values, text or numeric will be stored in the currently selected point/cell for the specified editable field array above. This is in development."/>
          </property>
          <property name="text">
           <string>Yes, No</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="imagoLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="imagoGroupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Imago Support</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QGridLayout" name="imagoGrid">
           <item row="0" column="0">
            <widget class="QLabel" name="imagoUserLabel">
             <property name="text">
              <string>UserID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="imagoPasswordLabel">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="imagoImageTypeLabel">
             <property name="text">
              <string>Image Type</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="imagoWorkspaceLabel">
             <property name="text">
              <string>Workspace</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="imagoImageryTypeLabel">
             <property name="text">
              <string>Imagery Type</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="imagoUserValue"/>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="imagoImageTypeCombo"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="imagoPasswordValue">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="imagoDatasetLabel">
             <property name="text">
              <string>Dataset</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="imagoImageryTypeCombo"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="imagoWorkspaceCombo"/>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="imagoDatasetCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="imagoLoginButton">
           <property name="text">
            <string>Login to Imago</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cHeader">
        <property name="text">
         <string>Collaboration</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QLineEdit" name="cPortValue">
          <property name="text">
           <string>5555</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="cServerLabel">
          <property name="toolTip">
           <string>The IP address or server name to connect to</string>
          </property>
          <property name="text">
           <string>Server</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="cSessionValue">
          <property name="text">
           <string>pv</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="cNameLabel">
          <property name="toolTip">
           <string>Your name to appear in VR</string>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="cSessionLabel">
          <property name="toolTip">
           <string>What session to connect to. A session is a string name. People who join the same session on the same server will see each other.</string>
          </property>
          <property name="text">
           <string>Session</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="cServerValue">
          <property name="text">
           <string>vrserver.kitware.com</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="cNameValue">
          <property name="text">
           <string>Anonymous</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="cPortLabel">
          <property name="toolTip">
           <string>What port to connect to</string>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="cConnectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="outputWindow"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>multisamples</tabstop>
  <tabstop>baseStationVisibility</tabstop>
  <tabstop>sendToOpenVRButton</tabstop>
  <tabstop>attachToCurrentViewButton</tabstop>
  <tabstop>showVRViewButton</tabstop>
  <tabstop>exportLocationsAsViewButton</tabstop>
  <tabstop>exportLocationsAsSkyboxesButton</tabstop>
  <tabstop>cropThickness</tabstop>
  <tabstop>editableField</tabstop>
  <tabstop>fieldValues</tabstop>
  <tabstop>imagoUserValue</tabstop>
  <tabstop>imagoPasswordValue</tabstop>
  <tabstop>imagoWorkspaceCombo</tabstop>
  <tabstop>imagoDatasetCombo</tabstop>
  <tabstop>imagoImageryTypeCombo</tabstop>
  <tabstop>imagoImageTypeCombo</tabstop>
  <tabstop>imagoLoginButton</tabstop>
  <tabstop>cServerValue</tabstop>
  <tabstop>cSessionValue</tabstop>
  <tabstop>cNameValue</tabstop>
  <tabstop>cPortValue</tabstop>
  <tabstop>cConnectButton</tabstop>
  <tabstop>outputWindow</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
